README.md

```diff
@@ -57,10 +57,14 @@ These scripts can be used to assist in setup of a brand new Windows Server as a
 ```powershell
 Set-ExecutionPolicy Bypass -Scope Process -Force
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::tls12
-$QuickStart = 'https://raw.githubusercontent.com/adilio/choco-quickstart-scripts/main/Start-C4BSetup.ps1'
-Invoke-Expression -Command ((New-Object System.Net.WebClient).DownloadString($QuickStart))
+Invoke-Expression -Command ((New-Object System.Net.WebClient).DownloadString('https://ch0.co/quickstart'))
 ```
 
+> :warning: **Warning**
```

Comment:
Might be better to be explicit here anyways? Short link is cool, but long way means everyone can use it and clearly see where code is coming from etc.

Reply:
Very Fair. I'll revert this back in the next commit.